pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -276,7 +276,7 @@ func (ctx *context) handleInboundInvitation(invitation *Invitation) error {
 	//  using this request's ID. issue-280
 	request := &Request{
 		Type:  ConnectionRequest,
-		ID:    uuid.New().String(),
+		ID:    ctx.thid,
```

Comment:
You need to pass thid as argument and remove thid from context

Reply:
ok i will do that way